packages/shared/src/components/Feed.spec.tsx

```diff
@@ -476,8 +476,11 @@ describe('Feed logged in', () => {
       },
     ]);
 
-    const [menuBtn] = await screen.findAllByLabelText('Options');
-    fireEvent.click(menuBtn);
+    const menuBtn = await screen.findByLabelText('Options');
+    fireEvent.keyDown(menuBtn, {
```

Comment:
Why do we have to fire this event now?

Reply:
@rebelchris there is `userEvent` which should trigger those intermediate events as well.

Reply:
@capJavert I know, but it's a full package now? seemed bit overkill as this already solved it.